docs/ui-extensions-sdk-frontend.md

```diff
@@ -22,29 +22,18 @@ communicate with the Contentful Management App.
 You will need to include the `contentful-extension-sdk` library in your HTML5 app like
 so:
 ~~~html
-<script src="https://contentful.github.io/ui-extensions-sdk/cf-extension-api.js"></script>
+<script src="https://unpkg.com/contentful-ui-extensions-sdk@2"></script>
```

Comment:
Problem with `@latest` is that introducing breaking changes would likely break existing UI extensions developed if they reference it directly. I'd prefer absolute version usage so that it's up to users to decide when and how they want to migrate.